src/qute/languageServer/client.ts

```diff
@@ -5,7 +5,7 @@ import { LanguageClient } from 'vscode-languageclient/node';
 import { JavaExtensionAPI } from '../../extension';
 import { sendTelemetry } from '../../utils/telemetryUtils';
 import { QuteClientCommandConstants } from '../commands/commandConstants';
-import { registerQuteLSDependentCommands, registerVSCodeQuteCommands, synchronizeQuteValidationButton } from '../commands/registerCommands';
+import { registerQuteLSDependentCommands, registerUpdateQuteLanguageIdCommand, registerVSCodeQuteCommands, synchronizeQuteValidationButton } from '../commands/registerCommands';
```

Comment:
VS Code is telling me that `registerUpdateQuteLanguageIdCommand` doesn't exist, I don't see it in `src/qute/commands/registerCommands.ts`

Reply:
My bad, I forget to push teh client.ts which remove this command. Sorry @datho7561  -(